### DISCUSSION TITLE
Multiple Plugin

### DISCUSSION THREAD

**Discussion by hirael** [tcgoetz/GarminDB#133] [Q&A] [answered]
Hello - Has anyone managed to work with multiple plugins in a single fit-file?

Best workaround I have is to edit file.py and change [0] to [1] and re-run the import.
     self.dev_application_id = self.dev_data_id[0].fields.application_id

thank you

**Comment by tcgoetz** [comment]
Fix incoming. Will require updating the Fitfile package.

**Comment by tcgoetz** [comment, marked as answer]
Fix is committed (92b9df3) to the develop branch. Please test and respond with success/fail. I will release new packages to PyPI after I get feedback.You will need to update your plugins in your HealthData/Plugins directory from the plugins repo.

**Comment by hirael** [comment]
Thank you - I'll give it a try over the wekend and report back.

**Reply by hirael** [reply]
Hi - This is probably because I cannot get the make-file to work on Windows, so I probably missed something during my manual update, but I had to change the plugin file import to the previous verion "from activity_plugin_base import ActivityPluginBase"

Otherwise the multiple plugin worked great thank you :)
